Documentation/guides.xml

```diff
@@ -14,6 +14,10 @@
         edit-on-github-branch="main"
         interlink-shortcode="t3docsRenderGuides"
         typo3-core-preferred="stable"
+        project-home="https://github.com/TYPO3-Documentation/render-guides"
```

Comment:
Until now, the attributes are sorted alphabetically. Can we stay that way?

Reply:
Ack